### ISSUE TITLE
missing key MapInsight

### ISSUE THREAD

**Issue by welpdx** [adxsoft/MapInsight-Addon#3]
Hello,
I was just installing the addon today. I kept receiving this error "missing key MapInsight"

<details>
        <summary>Full Freeplane Error</summary>

	at 2020-07-15 17:17:25 CMT,  hash c90402305ec6e3651d1c51a19806f877
	previous report : no reports sent before
	freeplane_version = 1.8.6; freeplane_xml_version = freeplane 1.8.0
	git revision = 19c2bc1bd55ec7a5a8e10362b39c0197720ff69b Dimitry Polivaev 2020-06-24T20:56:30Z
	java_version = 1.8.0_211; os_name = Windows 10; os_version = 10.0
	missing key MapInsight

</details>

Thanks,

Welp

**Comment by adxsoft** [comment]
Hi

Just managed to get hold of Windows 10 laptop with Freeplane 1.8.6 and can confirm this is an issue.

I’m not sure what the problem is as it installs in Freeplane 1.7 without any errors so I’ve put in a support request with Freeplane team to see if they can help with a solution.

I'll email you once I have it solved.

Regards
Allan @ adxsoft

**Comment by adxsoft** [comment]
Heres the forum post on the Freeplane support page.
https://sourceforge.net/p/freeplane/discussion/758437/thread/eb365be149/ <https://sourceforge.net/p/freeplane/discussion/758437/thread/eb365be149/>

Cheers
Allan

**Comment by adxsoft** [comment]
Freeplane 1.8 has issue with loading addons. I have changed the Mapinsight addon file as follows

`Replace <attribute NAME="menuTitleKey" VALUE="MapInsight"/> by <attribute NAME="menuTitleKey" VALUE="addons.MapInsight"/> in file MapInsight.addon.mm and its definition file MapInsight.mm`

I have tested this with Mac and Windows and seems to fix the Freeplane addon loading issue.

Hopefully Freeplane will correct this so all existing addons work.
